marketplace_builder/views/partials/layouts/_contributors.liquid

```diff
@@ -1 +1,23 @@
-<div class="contributors d-flex" data-contributors></div>
+{% query_graph "get_page_contributors", slug: context.location.href %}
+{% assign page_contributors = g.customizations.results.last %}
+
+{% if page_contributors != blank %}
+  <div class="contributors d-flex">
+    <p class="mb-0 d-flex align-items-center">
+      <span>Last edit:</span>&nbsp; {{ page_contributors.last_edit | date: "%b %d, %Y" }}
+    </p>
+    <p class="mb-0 d-flex align-items-center">
+      <span>Contributors:</span>&nbsp;
+      {% assign c = page_contributors.contributors | default: "[]" | to_hash | reverse %}
+      {% for contributor in c %}
+        {% if contributor.login != blank %}
+          <a href="https://github.com/{{ contributor.login }}" target="_blank">
+            <img src="{{ contributor.avatar_url }}" width="20" height="20" alt="{{ contributor.login }}">
```

Comment:
add `&s=20` to avatar url - it will download 20x20 image which will be much faster

https://github.com/mdyd-dev/nearme-documentation/blob/dcc2381be815d12c0e28a42c8b721cec58acf044/src/js/githubContributors.js#L18